auth/src/main/res/values/styles.xml

```diff
@@ -230,4 +230,12 @@
         <item name="android:textColor">#FFFFFFFF</item>
     </style>
 
+    <style name="FirebaseUI.TopProgressBar" parent="@style/Widget.MaterialProgressBar.ProgressBar.Horizontal">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">4dp</item>
+        <item name="android:indeterminate">true</item>
+        <item name="mpb_progressStyle">horizontal</item>
```

Comment:
I wish the Android team would fix style attributes, those look so dang ugly! 😭 Anyway, 👍